tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -133,7 +130,7 @@ async def test_hubspot_access_request_task(
 async def test_hubspot_erasure_request_task(
     db,
     policy,
-    erasure_policy_string_rewrite,
+    erasure_policy_string_rewrite_name_and_email,
```

Comment:
I updated the erasure policy to one that targets not only names but also the email address, this will let us verify the request override

Reply:
I definitely noticed this on the Braze one we did but I wasn't sure if it was specific or not - thank you for the explainer in here! 🙌🏽